### PR TITLE
Use new data and JS code

### PR DESCRIPTION
**Pull request by MonikaFu**
It seems that for some plots we were using outdated plot code and data. In this PR I update the code and relevant data so that we use the most recent versions.

### CODE REVIEW

**Comment by jdhoffa** [comment]
Hey @MonikaFu the Linter seems to be failing. 

Can you style the code, please? You can do so by running `prettier --write .` (`npm install -g prettier`)

**Review by jdhoffa — APPROVED** [review]
LGTM

I notice that the Linter is failing, but this is a larger topic and not directly related to this PR, so I opened a new issue #29 

I had a NB comment as well, that I will open another new issue for.

File: src/routes/sector_view.svelte [hunk 1]
```diff
@@ -107,10 +107,9 @@
 				<label class="label">
 					<span>Scenario</span>
 					<select class="select variant-outline-surface" id="scenario_selector">
-						<option value="ETP2017">ETP2017</option>
-						<option value="GECO2019">GECO2019</option>
-						<option value="WEO2019">WEO2019</option>
-						<option value="WEO2020">WEO2020</option>
+						<option value="GECO2023">GECO2023</option>
+						<option value="WEO2023">WEO2023</option>
+						<option value="ISF2023">ISF2023</option>
```
[the remark this answers is not in the document]
Comment (MonikaFu) [reply src/routes/sector_view.svelte:112]: yes, that is my idea, I created an ADO task around it: https://dev.azure.com/RMI-PACTA/2DegreesInvesting/_sprints/taskboard/P4I/2DegreesInvesting/2024_17?workitem=11679
But it is a task not for this PR.

[the remark this answers is not in the document]
Comment (jdhoffa) [reply src/routes/sector_view.svelte:112]: Totally agree :-) Just was flagging it

**Pull request merged**